### PR TITLE
For 7/9 release

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

**Comment by nibinqtl** [comment]
@burtonquantil @marcfishelquantil , is everything good enough to be released to public?

File: docs/edge-logic/built-in-variables.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ If you use the CDN360 portal to edit the property, the autocompletion feature wi
...
 | <span id="realtime_log_ds_factor"></span><span class="var">$realtime_log_ds_factor</span> |<span class="badge small" title="numerical value">#</span> RT log downsample factor. A value N means one log entry for every N requests| <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
-| <span id="realtime_log_ds_ratio"></span><span class="var">$realtime_log_ds_ratio</span> |<span class="badge small" title="numerical value">#</span> RT log downsample ratio. The value should be 1/N, as a decimal number between 0 and 1. (ETA: May 2021)| <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
+| <span id="realtime_log_ds_ratio"></span><span class="var">$realtime_log_ds_ratio</span> |<span class="badge small" title="numerical value">#</span> RT log downsample ratio. The value should be 1/N, as a decimal number between 0 and 1.| <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
```
Comment (burtonquantil) [inline docs/edge-logic/built-in-variables.md:43]: I'd just write "The value should be a decimal number between 0 and 1".

File: docs/edge-logic/built-in-variables.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ If you use the CDN360 portal to edit the property, the autocompletion feature wi
...
 | <span id="tcpinfo_min_rtt"></span><span class="var">$tcpinfo_min_rtt</span> |<span class="badge small" title="numerical value">#</span> minimum RTT, in microseconds, observed by TCP stack for the connection | <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
 | <span id="tcpinfo_rtt"></span><span class="var">$tcpinfo_rtt</span> |<span class="badge small" title="numerical value">#</span> latest RTT, in microseconds, observed by TCP stack for the connection | <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
-| <span id="time_http"></span><span class="var">$time_http</span> | current time in RFC7231 format that can be used for HTTP Date header (ETA: May 2021)| <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
+| <span id="time_http"></span><span class="var">$time_http</span> | current time in RFC7231 format that can be used for HTTP Date header | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (burtonquantil) [inline docs/edge-logic/built-in-variables.md:69]: I'd put a space between RFC and the number (i.e., "RFC 7231"), but it's minor.

File: docs/edge-logic/supported-directives.md [hunk 3]
```diff
@@ -810,6 +810,9 @@ proxy_cache_valid $cache_time;
 # extract a part from the origin's response header and send to client
 proxy_set $version_number $1 if($upstream_http_version ~ "Version:(.*)$");
 add_header version-number $version_number;
+# do not cache status codes 301 and 302 from the origin
+proxt_set $no_store 1 if ($upstream_response_status ~ 30[12]);
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:814]: proxt_set doesn't look like a directive. Do you mean "proxy_set" ?

**Review by burtonquantil — CHANGES_REQUESTED** [review]
@nibinqtl I think you intended to refer to "proxy_set"?

**Comment by marcfishelquantil** [comment]
I agree with Burton's suggestion and did not spot other issues.

File: docs/edge-logic/supported-directives.md (hunk 3 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: fixed.

**Review by burtonquantil — APPROVED** [review]
ok to merge

**Review by burtonquantil — APPROVED** [review]
Looks good.  I presume all the /en pages are just Lijo and Eleazar's tests which aren't yet exposed to the public.

**Pull request merged**